### ISSUE TITLE
LDAP SSO troubleshooting

### ISSUE THREAD

**Issue by afawaz2** [freescout-helpdesk/freescout#872]
I am attempting to use an HTTP header to enable the SSO feature. The header will contain the user email address. I tested by enabling the ldap sync and setting 

* $_SERVER Key: HTTP_XYZ
* Locate Users By: userprincipalname 

LDAP authentication is successful. Do you have any tips for troubleshooting?

**Comment by tiredofit** [comment]
I was the one who requested the feature but have also not been able to get it to work effectively. I believe I did once but I can't be sure.

What we did to bypass this was extend the app/Http/Controllers/Auth/LoginController.php with the Server variable lookup code there and it worked just fine. It shouldn't be in the main codebase though as I would call it dirty hack. We've been using the automatic login function this way for 3 months now with great success.
